src/components/app/App.tsx

```diff
@@ -0,0 +1,10 @@
+import { TaskListContainer } from "../taskListContainer/TaskListContainer";
+import "./App.css";
+
+export const App = () => {
+  return (
+    <div className="page-container">
+      <TaskListContainer />
+    </div>
+  );
+};
```

Comment:
I give you a hint of what the return type is for JSX.
```
export const App = (): ReactElement => (
  <div className="page-container">
    <TaskListContainer />
  </div>
);
```

Also, you can notice that you can skip the `return` if there is no logic happening between the `{}` and the `return`.
Of course, it works for anonymous functions and not for named functions.

Reply:
Oh yeah, cause anonymous will return implicitly, yeah?